### PR TITLE
Change to CargoIntake

### PR DESCRIPTION
**Pull request by TheHarmonicRealm**
I made `ARM_DOWNING` stop the motors

### CODE REVIEW

File: src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java [hunk 1]
```diff
@@ -206,6 +204,8 @@ public void onLoop(double timestamp)
                     /* Placing/intaking game pieces */
                     case INTAKING_AND_ALIGNING_CLOSEST_FORWARD_TARGET:
                         mCargoIntake.setWantedState(CargoIntake.WantedState.INTAKE);
+                        //Bring chute up to avoid collision
+                        mCargoChute.setWantedState(CargoChute.WantedState.SHOOT_ROCKET);
```
Comment (pietroglyph) [inline src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java]: Remove line 207 and 208.

File: src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java [hunk 2]
```diff
@@ -216,13 +216,15 @@ public void onLoop(double timestamp)
                         if (mWantedState == WantedState.ALIGN_AND_INTAKE_CARGO && (mDrive.isDoneWithTrajectory() || mCargoChute.atTarget()))
                         {
                             mCargoIntake.setWantedState(CargoIntake.WantedState.HOLD);
+                            mCargoChute.setWantedState(CargoChute.WantedState.SHOOT_BAY);
```
Comment (pietroglyph) [inline src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java]: This should be `CargoChute.WantedState.LOWER`.

File: src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java [hunk 3]
```diff
@@ -216,13 +216,15 @@ public void onLoop(double timestamp)
...
                             mDrive.setOpenLoop(DriveSignal.BRAKE);
                             mWantedState = WantedState.DRIVER_CONTROL;
                             newState = SystemState.DRIVER_CONTROLLING;
                         }
                         break;
                     case ALIGNING_CLOSEST_REVERSE_TARGET:
                         mCargoIntake.setWantedState(CargoIntake.WantedState.HOLD);
+                        mCargoChute.setWantedState(CargoChute.WantedState.SHOOT_BAY);
```
Comment (pietroglyph) [inline src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java]: This should be `CargoChute.WantedState.LOWER`.

File: src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java [hunk 4]
```diff
@@ -266,12 +268,20 @@ else if (mWantedState == WantedState.ALIGN_AND_SHOOT_CARGO_BAY
                         break;
                     case EJECTING_CARGO:
                         if (mWantedState == WantedState.ALIGN_AND_SHOOT_CARGO_BAY)
+                        {
                             mCargoChute.setWantedState(CargoChute.WantedState.SHOOT_BAY);
+                            //Arm *should* already be up, but this is a contingency
+                            mCargoIntake.setWantedState(CargoIntake.WantedState.HOLD);
```
Comment (pietroglyph) [inline src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java]: Remove this?

File: src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java [hunk 5]
```diff
@@ -266,12 +268,20 @@ else if (mWantedState == WantedState.ALIGN_AND_SHOOT_CARGO_BAY
...
+                        }
                         else if (mWantedState == WantedState.ALIGN_AND_SHOOT_CARGO_ROCKET)
+                        {
                             mCargoChute.setWantedState(CargoChute.WantedState.SHOOT_ROCKET);
-                        else
+                            //Brings arm down to avoid collision
+                            mCargoIntake.setWantedState(CargoIntake.WantedState.ARM_DOWN);
+                        }
+                        else {
```
Comment (j-james) [inline src/main/java/com/spartronics4915/frc2019/subsystems/Superstructure.java]: Remove the else brackets here to fit with our code conventions

**Pull request merged**